eos/form-factors/mesonic-impl.hh

```diff
@@ -53,6 +54,25 @@ namespace eos
 
     /* P -> V Processes */
 
+    struct BToDstar {
+        static constexpr const char * label = "B->D^*";
+        static constexpr double mB = 5.279;
+        static constexpr double mV = 2.0103;
+        static constexpr double mBc = 6.2751;
+        static constexpr double mR2_0m = (mBc + 0.000) * (mBc + 0.000);
+        static constexpr double mR2_1m = (mBc + 0.056) * (mBc + 0.056);
+        static constexpr double mR2_1p = (mBc + 0.492) * (mBc + 0.492);
+    };
+
+    struct BToRho {
```

Comment:
Can you move the ```BToRho``` class below ```BToKstar```, such that we keep the classes orderd alphabetically? It helps finding things later on.